lnwallet/chancloser/rbf_coop_test.go

```diff
@@ -830,6 +1001,238 @@ func newCloser(t *testing.T, cfg *harnessCfg) *rbfCloserTestHarness {
 	return chanCloser
 }
 
+// testInitiatorShutdownRecvOk is a helper function that tests the initiator
+// shutdown received scenario for both taproot and non-taproot channels in the
+// ShutdownPending state.
+func testInitiatorShutdownRecvOk(t *testing.T, ctx context.Context,
```

Comment:
I think all these test helpers can be broken into two - one for non tap and one for tap, and we perform the branch switch at the callsite, such that it's easier to read and maintain.

Reply:
Makes sense, will update.

Reply:
Pushed as a fix up commit, let me know if this is what you had in mind.